CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.7.7] - 2021-06-01
```

Comment:
The change note should be under [Unreleased]  The version will be updated when the PR is merged.
Ex:
## [Unreleased]
### Fixed
- User logged and then logout, the button is not shown as activated.
- ## [1.7.6] - 2020-05-28

Then when merging the PR, choose Patch